pkg/ceph/client/image.go

```diff
@@ -44,17 +45,17 @@ func ListImages(context *clusterd.Context, clusterName, poolName string) ([]Ceph
 
 	//The regex expression captures the json result at the end buf
 	//When logLevel is DEBUG buf contains log statements of librados (see tests for examples)
-	res := regexp.MustCompile(`\[(.*)\]$`).FindStringSubmatch(string(buf))
+	//It can happen that the end of the "real" output doesn't not contain a new line
+	//that's why looking for the end isn't an option here (anymore?)
+	res := regexp.MustCompile(`(?m)^\[(.*)\]`).FindStringSubmatch(string(buf))
```

Comment:
The block storage test failed because the ceph/rbd output was beginning at the line but not ending with a new line. This changed regex made the test find the output again.